unstructured/partition/pptx.py

```diff
@@ -284,7 +304,7 @@ def iter_shapes(shapes: _BaseGroupShapes) -> Iterator[BaseShape]:
         def sort_key(shape: BaseShape) -> Tuple[int, int]:
             return shape.top or 0, shape.left or 0
 
-        return sorted(iter_shapes(slide.shapes), key=sort_key)
+        return sorted(iter_shapes(slide.shapes), key=sort_key), getattr(slide.shapes, "title", None)
```

Comment:
`Shapes.title` is always present, isn't it? Just that it returns `() -> Shape | None`? So no need for a `getattr()` here, just `slide.shapes.title`.

Reply:
Ohhh, right, of course. There's no entry for `Shapes.title` in the type-stub yet.

You can add this to the bottom of `typings/pptx/shapes/shapetree.pyi` to fix that up:
```python
    @property
    def title(self) -> Shape | None: ...
```

Will look like this afterward and that should get rid of the mypy error:
```python
class SlideShapes(_BaseGroupShapes):
    def __iter__(self) -> Iterator[BaseShape]: ...
    @property
    def title(self) -> Shape | None: ...
```